### PR TITLE
chore: Fix linter findings for Windows (part3)

### PR DESCRIPTION
**Pull request by zak-pawel**
It is only part of the bigger job: https://github.com/influxdata/telegraf/issues/13036
After all findings in whole project are handled for `Windows`, linter CI job for Windows can be enabled.

Fixes for following findings in `plugins/inputs/win_perf_counters/...` have been made:
```
plugins/inputs/win_perf_counters/kernel32.go:40:2                              unused  field `wYear` is unused
plugins/inputs/win_perf_counters/kernel32.go:41:2                              unused  field `wMonth` is unused
plugins/inputs/win_perf_counters/kernel32.go:42:2                              unused  field `wDayOfWeek` is unused
plugins/inputs/win_perf_counters/kernel32.go:43:2                              unused  field `wDay` is unused
plugins/inputs/win_perf_counters/kernel32.go:44:2                              unused  field `wHour` is unused
plugins/inputs/win_perf_counters/kernel32.go:45:2                              unused  field `wMinute` is unused
plugins/inputs/win_perf_counters/kernel32.go:46:2                              unused  field `wSecond` is unused
plugins/inputs/win_perf_counters/kernel32.go:47:2                              unused  field `wMilliseconds` is unused
plugins/inputs/win_perf_counters/kernel32.go:60:2                              unused  var `krn_FileTimeToSystemTime` is unused
plugins/inputs/win_perf_counters/kernel32.go:60:2                              revive  var-naming: don't use underscores in Go names; var krn_FileTimeToSystemTime should be krnFileTimeToSystemTime
plugins/inputs/win_perf_counters/kernel32.go:61:2                              revive  var-naming: don't use underscores in Go names; var krn_FileTimeToLocalFileTime should be krnFileTimeToLocalFileTime
plugins/inputs/win_perf_counters/kernel32.go:61:2                              unused  var `krn_FileTimeToLocalFileTime` is unused
plugins/inputs/win_perf_counters/kernel32.go:62:2                              revive  var-naming: don't use underscores in Go names; var krn_LocalFileTimeToFileTime should be krnLocalFileTimeToFileTime
plugins/inputs/win_perf_counters/kernel32.go:63:2                              unused  var `krn_WideCharToMultiByte` is unused
plugins/inputs/win_perf_counters/kernel32.go:63:2                              revive  var-naming: don't use underscores in Go names; var krn_WideCharToMultiByte should be krnWideCharToMultiByte
plugins/inputs/win_perf_counters/pdh.go:47:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:48:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:49:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:50:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:60:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:61:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:62:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:63:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:64:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:65:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:66:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:67:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:68:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:69:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:70:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:71:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:72:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:73:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:74:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:75:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:76:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:77:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:78:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:79:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:80:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:81:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:82:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:83:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:84:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:85:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:86:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:87:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:88:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:89:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:90:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:91:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:92:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:93:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:94:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:95:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:96:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:97:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:98:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:99:2                                   revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:100:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:101:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:102:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:103:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:104:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:105:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:106:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:107:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:108:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:109:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:110:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:111:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:112:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:113:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:114:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:115:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:116:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:117:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:118:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:119:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:120:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:121:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:122:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:123:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:124:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:125:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:126:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:127:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:128:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:129:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:130:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:131:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:132:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:133:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:134:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:135:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:136:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:137:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:138:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:139:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:140:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:141:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:142:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:143:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:144:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:145:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:239:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:240:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:241:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:242:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:243:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:244:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:245:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:246:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:247:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:248:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:249:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:250:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:254:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:255:2                                  revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh.go:263:2                                  revive  var-naming: don't use underscores in Go names; var pdh_AddCounterW should be pdhAddCounterW
plugins/inputs/win_perf_counters/pdh.go:264:2                                  revive  var-naming: don't use underscores in Go names; var pdh_AddEnglishCounterW should be pdhAddEnglishCounterW
plugins/inputs/win_perf_counters/pdh.go:265:2                                  revive  var-naming: don't use underscores in Go names; var pdh_CloseQuery should be pdhCloseQuery
plugins/inputs/win_perf_counters/pdh.go:266:2                                  revive  var-naming: don't use underscores in Go names; var pdh_CollectQueryData should be pdhCollectQueryData
plugins/inputs/win_perf_counters/pdh.go:267:2                                  revive  var-naming: don't use underscores in Go names; var pdh_CollectQueryDataWithTime should be pdhCollectQueryDataWithTime
plugins/inputs/win_perf_counters/pdh.go:268:2                                  revive  var-naming: don't use underscores in Go names; var pdh_GetFormattedCounterValue should be pdhGetFormattedCounterValue
plugins/inputs/win_perf_counters/pdh.go:269:2                                  revive  var-naming: don't use underscores in Go names; var pdh_GetFormattedCounterArrayW should be pdhGetFormattedCounterArrayW
plugins/inputs/win_perf_counters/pdh.go:270:2                                  revive  var-naming: don't use underscores in Go names; var pdh_OpenQuery should be pdhOpenQuery
plugins/inputs/win_perf_counters/pdh.go:271:2                                  revive  var-naming: don't use underscores in Go names; var pdh_ValidatePathW should be pdhValidatePathW
plugins/inputs/win_perf_counters/pdh.go:272:2                                  revive  var-naming: don't use underscores in Go names; var pdh_ExpandWildCardPathW should be pdhExpandWildCardPathW
plugins/inputs/win_perf_counters/pdh.go:273:2                                  revive  var-naming: don't use underscores in Go names; var pdh_GetCounterInfoW should be pdhGetCounterInfoW
plugins/inputs/win_perf_counters/pdh.go:274:2                                  revive  var-naming: don't use underscores in Go names; var pdh_GetRawCounterValue should be pdhGetRawCounterValue
plugins/inputs/win_perf_counters/pdh.go:275:2                                  revive  var-naming: don't use underscores in Go names; var pdh_GetRawCounterArrayW should be pdhGetRawCounterArrayW
plugins/inputs/win_perf_counters/pdh.go:567:21                                 revive  var-naming: func parameter msgId should be msgID
plugins/inputs/win_perf_counters/pdh_amd64.go:36:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:42:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:48:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:51:2                             unused  field `padding` is unused
plugins/inputs/win_perf_counters/pdh_amd64.go:55:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:61:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:67:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:73:6                             revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:120:6                            revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/pdh_amd64.go:135:6                            revive  var-naming: don't use ALL_CAPS in Go names; use CamelCase
plugins/inputs/win_perf_counters/performance_query.go:165:2                    revive  early-return: if c { ... } else { ... return } can be simplified to if !c { ... return } ...
plugins/inputs/win_perf_counters/performance_query.go:168:10                   revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block
plugins/inputs/win_perf_counters/performance_query.go:251:89                   revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/performance_query.go:260:2                    revive  early-return: if c { ... } else { ... return } can be simplified to if !c { ... return } ...
plugins/inputs/win_perf_counters/performance_query.go:263:10                   revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block
plugins/inputs/win_perf_counters/win_perf_counters.go:91:1                     revive  function-result-limit: maximum number of return results per function exceeded; max 3 but got 5
plugins/inputs/win_perf_counters/win_perf_counters.go:91:1                     revive  confusing-results: unnamed results of the same type may be confusing, consider using named results
plugins/inputs/win_perf_counters/win_perf_counters.go:168:1                    revive  argument-limit: maximum number of arguments per function exceeded; max 6 but got 9
plugins/inputs/win_perf_counters/win_perf_counters.go:191:1                    revive  argument-limit: maximum number of arguments per function exceeded; max 6 but got 8
plugins/inputs/win_perf_counters/win_perf_counters.go:465:117                  revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters.go:502:41                   revive  empty-lines: extra empty line at the start of a block
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:17:59   revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:144:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:182:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:236:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:289:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:342:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:393:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:436:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:484:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:522:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:561:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:584:59  revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:600:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:633:59  revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:651:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:689:61  revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_integration_test.go:705:2   revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_test.go:73:9                revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
plugins/inputs/win_perf_counters/win_perf_counters_test.go:84:9                revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
plugins/inputs/win_perf_counters/win_perf_counters_test.go:101:9               revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
plugins/inputs/win_perf_counters/win_perf_counters_test.go:145:32              unused  func `(*FakePerformanceQuery).findCounterByHandle` is unused
plugins/inputs/win_perf_counters/win_perf_counters_test.go:164:6               revive  early-return: if c { ... } else { ... return } can be simplified to if !c { ... return } ...
plugins/inputs/win_perf_counters/win_perf_counters_test.go:174:11              revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
plugins/inputs/win_perf_counters/win_perf_counters_test.go:192:6               revive  early-return: if c { ... } else { ... return } can be simplified to if !c { ... return } ...
plugins/inputs/win_perf_counters/win_perf_counters_test.go:202:11              revive  indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)
plugins/inputs/win_perf_counters/win_perf_counters_test.go:241:1               revive  argument-limit: maximum number of arguments per function exceeded; max 6 but got 8
plugins/inputs/win_perf_counters/win_perf_counters_test.go:251:2               revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_test.go:436:46              revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_test.go:676:73              revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_test.go:821:67              revive  empty-lines: extra empty line at the end of a block
plugins/inputs/win_perf_counters/win_perf_counters_test.go:824:2               revive  unexported-naming: the symbol PerfObject is local, its name should start with a lowercase letter
plugins/inputs/win_perf_counters/win_perf_counters_test.go:964:45              revive  empty-lines: extra empty line at the start of a block
plugins/inputs/win_perf_counters/win_perf_counters_test.go:1452:54             revive  empty-lines: extra empty line at the end of a block
```

### CODE REVIEW

File: plugins/inputs/win_perf_counters/pdh_arm64.go [hunk 1]
```diff
@@ -32,47 +32,48 @@
...
 
-// PDH_COUNTER_INFO structure contains information describing the properties of a counter. This information also includes the counter path.
-type PDH_COUNTER_INFO struct {
+// PdhCounterInfo structure contains information describing the properties of a counter. This information also includes the counter path.
+type PdhCounterInfo struct {
 	//Size of the structure, including the appended strings, in bytes.
 	DwLength uint32
-	//Counter type. For a list of counter types, see the Counter Types section of the <a "href=http://go.microsoft.com/fwlink/p/?linkid=84422">Windows Server 2003 Deployment Kit</a>.
+	//Counter type. For a list of counter types, see the Counter Types section
+	//of the <a "href=http://go.microsoft.com/fwlink/p/?linkid=84422">Windows Server 2003 Deployment Kit</a>.
```
Comment (srebhan) [inline plugins/inputs/win_perf_counters/pdh_arm64.go:76]:
```suggestion
	//of the Windows Server 2003 Deployment Kit (http://go.microsoft.com/fwlink/p/?linkid=84422).
```

**Review by srebhan — COMMENTED** [review]
LGTM. Just one small suggestion to avoid HTML tags in comments.

**Review by srebhan — APPROVED** [review]
Thanks @zak-pawel!

**Review by powersj — APPROVED** [review]
Thank you This PR scares me given the amount of code removed :\ It does look like a lot of the pdh/kernel32 code is from somewhere too. makes me wonder if we will ever need to make updates to it if this will make it harder, but we can +1 and deal with it then :D

**Pull request merged**